### ISSUE TITLE
Builds doc sentence needs fixing

### ISSUE THREAD

**Issue by etsauer** [rhtconsulting/openshift-playbooks#32]
The first sentence under [this section](https://github.com/rhtconsulting/openshift-playbooks/blob/master/playbooks/AppDev/builds.adoc#s2i-scripts) reads:

```
The following table provides an overview of the S2I builder image is required to implement the following scripts
```

This looks like two sentences crammed together?

/cc @sabre1041

**Comment by etsauer** [comment]
Additionally, "python" is listed twice in this list: https://github.com/rhtconsulting/openshift-playbooks/blob/master/playbooks/AppDev/builds.adoc#existing-builder-images

**Comment by sabre1041** [comment]
Will tackle along with #18 in the next few days in preparation for upcoming december enablement

**Comment by sabre1041** [comment]
@etsauer this issue was addressed in #46 so it can be closed
